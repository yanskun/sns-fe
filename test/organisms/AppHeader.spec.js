import { createLocalVue, shallowMount } from '@vue/test-utils'
import VueCompositionApi from '@vue/composition-api'
import Vuetify from 'vuetify'
import Component from '~/components/organisms/AppHeader'

const localVue = createLocalVue()
localVue.use(VueCompositionApi)
localVue.use(Vuetify)

describe('AppHeader', () => {
  test('snapshot', () => {
    const wrapper = shallowMount(Component, {
      propsData: {},
      localVue,
    })
    expect(wrapper.element).toMatchSnapshot()
  })
})
