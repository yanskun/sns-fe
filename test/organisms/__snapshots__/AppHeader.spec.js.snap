// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppHeader snapshot 1`] = `
<v-toolbar-stub
  extensionheight="48"
  src=""
  tag="header"
>
  <v-spacer-stub />
  <div
    class="tabs"
  >
    <div
      class="tab"
    >
      <v-btn-stub
        activeclass=""
        tag="button"
        type="button"
      >
        <v-icon-stub
          left="left"
        >
          mdi-home
        </v-icon-stub>
        Home
      </v-btn-stub>
    </div>
  </div>
  <v-spacer-stub />
  <v-icon-stub
    color="green darken-2"
    large="large"
  >
    mdi-account-circle
  </v-icon-stub>
</v-toolbar-stub>
`;
